### PR TITLE
bpo-37216: Fix filename in mac using document

### PR DESCRIPTION
**Pull request by MakDon**
The python folder name is wrong in mac using document. 

$ ls /Applications | grep Python
Python 2.7
Python 3.5
Python 3.6
Python 3.7

And my system version is macOS Mojave Version 10.14.5(18F132)
$uname -v
Darwin Kernel Version 18.6.0: Thu Apr 25 23:16:27 PDT 2019; root:xnu-4903.261.4~2/RELEASE_X86_64

This commit should be backport to 2.7, 3.5, 3.6. 
DO NOT backport to 3.7 and 3.8 because i  create another PR to fix the filename and version in branch 3.7 and 3.8.
See: 
[#13963 for 3.7](https://github.com/python/cpython/pull/13963), 
[#13964 for 3.8](https://github.com/python/cpython/pull/13964)
[#13966 for 3.9](https://github.com/python/cpython/pull/13966)

https://bugs.python.org/issue37216

### CODE REVIEW

**Comment by MakDon** [comment]
I could not find Build Applet mentioned on [distributing-python-applications-on-the-mac)](https://docs.python.org/3.7/using/mac.html#distributing-python-applications-on-the-mac). It seems that is has been removed.

**Comment by MakDon** [comment]
The document of Build Applet on [Using Python on a Macintosh](https://docs.python.org/3.7/using/mac.html#distributing-python-applications-on-the-mac) should be another issue. I will create a new issue for it if i could find something about it.

**Comment by ned-deily** [comment]
@MakDon, thanks for the PRs and for noticing the spurious reference to Build Applet, a tool that was removed in Python 3.  The whole "Using Python on a Macintosh" chapter has not been updated in a long time and is woefully out-of-date.  Despite good intentions, I have not yet gotten around to doing it but I will do so shortly - really.  I'm going to push the PRs you have already made but I would ask you to hold off on making further changes as major rewrites are necessary.  Thanks!

File: Doc/using/mac.rst
Comment (ned-deily) [inline Doc/using/mac.rst:28]: Should be `Python 3.9` for the master branch.

**Review by ned-deily — CHANGES_REQUESTED** [review]

File: Doc/using/mac.rst
Comment (MakDon) [reply Doc/using/mac.rst:28]: I only edit the folder name cause this commit need to be backported to branch 2.7, 3.5, 3.6

Comment (MakDon) [reply Doc/using/mac.rst]: Should i make another PR for updating this to 3.8 on master branch?

Comment (MakDon) [reply Doc/using/mac.rst:28]: Hi @ned-deily , i made a PR for update the version on master branch. See [13966](https://github.com/python/cpython/pull/13966). Thanks for your review!

**Comment by MakDon** [comment]
I have made the requested changes; please review again.
Thanks bot :D

**Comment by ned-deily** [comment]
It looks like your newer PR,  #13966, effectively replaces this PR so I"m just going to close this one and merge the other.  Thanks!

**Comment by MakDon** [comment]
@ned-deily hummm, the problem is, the folder name in [2.7](https://docs.python.org/2.7/using/mac.html#getting-and-installing-macpython), [3.5](https://docs.python.org/3.5/using/mac.html#getting-and-installing-macpython), [3.6](https://docs.python.org/3.6/using/mac.html#getting-and-installing-macpython) is wrong so i create this pr for the folder name.

**Comment by MakDon** [comment]
But it seems that it would be conflict if backport directly. Is it the best way is create PR for each version?

**Comment by MakDon** [comment]
For 

>  I would ask you to hold off on making further changes as major rewrites are necessary.

i would not create new PR the the floder name issue. Thanks for your review! :D

**Comment by ned-deily** [comment]
> But it seems that it would be conflict if backport directly. Is it the best way is create PR for each version?

The change in this case is a bit unusual in that the changed lines include the version number in them so an automated backport is not possible and someone, either the PR submitter or the core developer merging the original PR, would need to manually create the backports.  In general, it's best to just produce the PR for the master branch and then once it has been reviewed and accepted then produce any needed backport PRs if the core developer hasn't already done so.

**Comment by MakDon** [comment]
> > But it seems that it would be conflict if backport directly. Is it the best way is create PR for each version?
> 
> The change in this case is a bit unusual in that the changed lines include the version number in them so an automated backport is not possible and someone, either the PR submitter or the core developer merging the original PR, would need to manually create the backports. In general, it's best to just produce the PR for the master branch and then once it has been reviewed and accepted then produce any needed backport PRs if the core developer hasn't already done so.

Thanks for your description. :D

**Pull request closed without merging**
